### DISCUSSION TITLE
Regarding Final Evaluations

### DISCUSSION THREAD

**Discussion by Cypre55** [larics/icuas22_competition#63] [Q&A] [answered]
Hey!

I have a few queries regarding the final evaluation procedure. 
Will there be a new evaluation world separate from the one used currently?
Will the best run out of 5 be considered?

**Comment by fpetric** [comment, marked as answer]
Hi @Cypre55, 

the evaluation world will remain the one  that we are using now (note that this is not any of the worlds we provided). In the initial version of the rulebook we did say that the score will be the best of 5 runs, but in the most recent update of the rulebook (published on website on April 22nd I believe, and we announced it here even earlier) we said average of 5 best runs from the evaluation period. We will run the final upload 5 times, and if those would be the 5 best scores that a team achieves, we will average those. If a team has a better run from before, we will use that/those instead of the worse runs from the "final 5".

**Reply by afaroo01** [reply]
@fpetric, I see the ranking are displayed now for only final scores. Initially it was also for arena 3 getting to know what performance was achieved in a known arena compared to an unknown. Will that be displayed again, it was a good option?

**Reply by fpetric** [reply]
If we have the time, we will do arena 3 too.

**Reply by afaroo01** [reply]
@fpetric will the final evaluation be the same as before the deadline or the starting points and tag positions will be changed for the final 5 runs?

**Reply by fpetric** [reply]
As stated in the rulebook:
_The structure of the arena (size of the zones) will be the same as in the example files
shared with teams, but the layout and number of the obstacles, UAV take-off spot, and the
location of the target may vary._

and as answered [here](https://github.com/larics/icuas22_competition/discussions/56#discussioncomment-2702944), we will run the evaluation with different takeoff spots and different tag positions. These positions will be the same for all teams, meaning that we will select 5 takeoff positions (A, B, C, D, E) and 5 tag positions (a, b, c, d, e) and run each solution with the same pairs of takeoff and tag positions (Aa, Bb, Cc, Dd, Ee). And each team will get one run for each pair of takeoff and tag positions. All of these runs will be performed on the evaluation arena that we used from the start of the evaluation period.

**Reply by afaroo01** [reply]
Thanks @fpetric, its very clear now. And the average will be taken from the best 5 runs including from the scores got in the evaluation period?
